dotcom-rendering/src/lib/format.ts

```diff
@@ -149,3 +149,21 @@ export const formatToFEFormat = ({
 	display: displayToFEDisplay(display),
 	theme: themeToFETheme(theme),
 });
+
+/**
+ * Creates a string representation of {@linkcode ArticleFormat}. Useful for
+ * logging, storybook UI etc.
+ *
+ * @param format An {@linkcode ArticleFormat}
+ * @returns A string representation of `ArticleFormat`
+ */
+export const formatToString = ({
```

Comment:
I'm sure we've got another one of these somewhere… maybe in the formats decorators?